packages/zowe-explorer/src/extension.ts

```diff
@@ -71,6 +72,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<ZoweEx
  * @export
  */
 export async function deactivate() {
+    await ZoweSaveQueue.all();
```

Comment:
Should we add a test to verify that this method gets called on deactivate?

Reply:
thanks for adding the test 🥳